server/app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ApplicationRecord
+ 
```

Comment:
I think the models are just used to define relationships between resources 1:M, 1:1. Also used to validate data.  Because I have only one resource and haven't validated any data there is nothing entered here.

Reply:
thanks for the explanation